website/docs/Examples/Video_Experiment.md

```diff
@@ -1,8 +1,8 @@
 ---
 id: VideoExperiment
-title: Puppies and Kittens Experiment
+title: Trigger Addition
 ---
-# OpenBCI_Experiment
+# Trigger Addition
```

Comment:
This should be reverted back, because I barely understand what this means and I was around when the project was created.